keycloakservers.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- hosts: letsencrypt
+  roles:
+    - letsencrypt
+  remote_user: "debian"
+
+- hosts: proxy
+  roles:
+    - nginx
+  remote_user: "debian"
+
+- hosts: db
+  roles:
+    - postgresql
+  remote_user: "debian"
```

Comment:
these can be removed since they will be handled in other role-specif playbooks, e.g. dbservers.yml etc

Reply:
But, these are tightly coupled with the keycloak installation. 
How do you believe we could split them?

Reply:
Yes, will do that.